api/config/routes.rb

```diff
@@ -131,6 +131,7 @@
         resource :cart, controller: :cart, only: %i[show create] do
           post   :add_item
           patch  :empty
+          delete :delete
```

Comment:
```suggestion
        resource :cart, controller: :cart, only: %i[show create destroy] do
          post   :add_item
          patch  :empty
```

Reply:
Ah, I missed that one...
Anyway, have just adjusted changes, done.